reference/supported-languages.md

```diff
@@ -0,0 +1,30 @@
+# Languages/Formats OpenRewrite supports
+
+OpenRewrite is continuously adding support for new languages and formats. As of June 2023, OpenRewrite supports:
+
+## Programming languages
+
+* Java
+* Groovy/Gradle 
+* Kotlin
+* COBOL (Closed source, commercial feature)
+
+## Data formats
+
+* XML
+* Properties
+* YAML
+* JSON
+* Protobuf
+* Maven POMs (with dependency management)
+
+## Frameworks
+
+Framework developers have created recipes to help migrate and support their frameworks. These include:
+
+* Spring
+* Quarkus
+* Micronaut
+* Kubernetes
+
+You can find all of the frameworks in the [community marketplace](https://public.moderne.io/marketplace).
```

Comment:
Since these are the OSS docs, should we not link to the recipe catalog instead of the marketplace? Both contain the same, and while the marketplace looks better, it might still be slightly confusing for readers of the OSS docs to be transported elsewhere.
```suggestion
You can find all of the frameworks in the [recipe catalog](https://docs.openrewrite.org/recipes).
```

Reply:
Olga had mentioned that she wanted to call out the marketplace -- but I think there may just be some context missing. Agree that it's probably better to just link to the recipe catalog.